sfm/ui/templates/ui/collection_detail.html

```diff
@@ -192,7 +193,7 @@ <h4 class="modal-title" id="addNoteTurnOffModalLabel">Add note & turn off</h4>
 <div class="row">
     <div class="col-md-12">
       <div class="panel panel-default">
-        <div class="panel-heading"><h4>Seeds</h4></div>
+        <div class="panel-heading"><h4>Seeds<a href="{% url "download" collection.pk %}"><span class="pull-right glyphicon glyphicon-download-alt"></span></a></h4></div>
```

Comment:
Per @lwrubel, please make this a button with a text label like “Download seed list”.